pkg/meta/redis.go

```diff
@@ -605,6 +638,14 @@ func (m *redisMeta) totalInodesKey() string {
 	return m.prefix + totalInodes
 }
 
+func (m *redisMeta) aclKey(id uint32) string {
+	if key, ok := m.aclKeyCache[id]; ok {
+		return key
+	}
+	m.aclKeyCache[id] = fmt.Sprintf("%sacl%d", m.prefix, id)
```

Comment:
this is not thread-safe. I think we don't need this.

Reply:
I forgot to handle thread-safety here, and I switched it to sync.Map.